test/uiauto/base.js

```diff
@@ -72,40 +66,39 @@ async function newInstruments (bootstrapFile) {
 async function init (bootstrapFile, sock) {
   let proxy = new UIAutoClient(sock);
   let instruments = await newInstruments(bootstrapFile);
-  instruments.start(null, async () => {
-    await proxy.safeShutdown();
-    throw new Error('Unexpected shutdown of instruments');
+  instruments.on('exit', () => {
+    Promise.resolve(async () => {
```

Comment:
I don't understand. doesn't having `.done()` here mean that the error is thrown in the context of the exit handler? won't that result in an unhandled error?